shepherd-client/src/test/java/com/devsda/platform/shepherdclient/impl/ShepherdClientTest.java

```diff
@@ -94,4 +95,22 @@ public void executeEndpointTest() throws Exception {
         ObjectMapper objectMapper = new ObjectMapper();
         System.out.println(objectMapper.writeValueAsString(executeEndpointResponse));
     }
+
+    @Test public void saveDocumentTest() {
```

Comment:
+1 to see UTs for Document sharing usecase.

Reply:
Alas this is also removed.